lib/travis/api/v3/services/build/priority.rb

```diff
@@ -0,0 +1,9 @@
+module Travis::API::V3
+  class Services::Build::Priority < Service
+
+    def run
+      build = check_login_and_find(:build)
```

Comment:
Hey @dmaity-celestialsys, I don't see a query that actually changes the build priority. Are you going to add it later?

Reply:
Hi @pavel-d, Yes, I have covered query section inside different PR.

Reply:
Got it